dev/src/transaction.ts

```diff
@@ -433,30 +433,23 @@ export function parseGetAllArguments(
   let documents: DocumentReference[];
```

Comment:
I can't comment on unchanged code, but does the getAll() parameter definition need to be updated somehow?
```
   * @param {Array.<DocumentReference|ReadOptions>} documentRefsOrReadOptions
   * The `DocumentReferences` to receive, followed by an optional field
   * mask.
```